applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_yield_surface.cpp

```diff
@@ -0,0 +1,53 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Mohamed Nabi
+//
+
+#include "custom_constitutive/coulomb_yield_surface.hpp"
+#include "custom_constitutive/tension_cutoff.hpp"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(TestCoulombYieldSurface, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    double friction_angle = 45.0 * Globals::Pi / 180.0;
```

Comment:
The following function expresses your intent more directly:
```suggestion
    const auto friction_angle = MathUtils<>::DegreesToRadians(45.0);
```

Reply:
Could you please use `const auto` here rather than `double`?